draft-demarco-oauth-status-attestations.md

```diff
@@ -260,15 +260,17 @@ POST /status HTTP/1.1
 Host: issuer.example.org
 Content-Type: application/x-www-form-urlencoded
```

Comment:
I wonder if this should be changed to JSON.

If it says like this, I think an example showing application/x-www-form-urlencoded of CWT and JWT would help communicate the intention here, I am happy to produce one if there is agreement.

Reply:
I'm also in favor of `application/json` too

@marinaado @fmarino-ipzs ^